### ISSUE TITLE
Figure supplements

### ISSUE THREAD

**Issue by JGilbert-eLife** [substance/texture#1281]
# Description

eLife accommodates supplementary images that authors want to include as direct supplements to main figures. To quote the author guide, figure supplements “must be linked to one of the primary figures: they can, for example, provide additional examples of analyses or data shown in a primary figure.” There are no limits on how many figure supplements can be appended to one main figure. 

# User stories
### Author
- As an author, I want to be able to clearly see my figure supplements, so that I can be sure that none are missing and that their quality is sufficient.
- As an author, I want to be able to clearly see the relationship between a figure supplement and the main figure with which it is associated so that I can check this has been captured correctly.
- As an author, I want to be able to update the image for a figure supplement, so that I can fix errors in the images that were originally supplied.
- As an author, I want to be able to edit the title and legend for a figure supplement so that I can correct any errors during proofing.
- As an author, I want to be able to add a reference or asset citation to a figure supplement title and legend so that I can cite other articles or assets in this context.
- As an author, I want to be able to clearly identify any source data or code that is directly associated with a figure supplement so that I can confirm this relationship has been captured correctly.

### Production staff
- As production staff, I want to be able to add a new figure supplement to a main figure on behalf of the author, so that I can ensure that manuscript is complete for publishing.
- As production staff, I want to be able to remove an existing figure supplement on behalf of the author, so that the manuscript is up to date for publishing.
- As production staff, I want to know why an author has changed a figure supplement image so that I know reasons for the change.
- As production staff, I want to provide an explanation for why I am replacing a figure supplement image so that I can record the reason for the change.
- As production staff, I want to compare replacement a figure supplement image with the original so that I can check it matches the author's reason for the change.

# But what if . . . ?

- An author wants to cite Figure 3—figure supplement 1A? The link in the text should go to the figure, it does not need to go to part A, but part A needs to be part of the link. See #33 

# Consideration

- The labelling pattern is as, for example, follows:
  - Figure 1
    - Figure 1—figure supplement 1
    - Figure 1—figure supplement 2
  - Figure 2
    - Figure 2—figure supplement 1
  - Figure 3
    - Figure 3—figure supplement 1
    - Figure 3—figure supplement 2
    - Figure 3—figure supplement 3
- In most respects, figure supplements have all the same properties as main figures.
- Figure supplements can have source data, just like main figures.
- The label for a figure supplement should derive from the main figure with which it is associated. E.g. Figure 1 => Figure 1—figure supplement 1, Appendix 1—figure 1 => Appendix 1—figure 1—figure supplement 1.
- eLife supports figure supplements as an alternative to putting images in supplementary material. 

# XML requirements

A figure supplement has to be captured as a child of `fig-group` (the same `fig-group` as the corresponding main `fig`). The `fig` has the attribute `specific-use="child-fig"`, which designates it as a figure supplement.
```xml
 <fig-group>
         ...
          <fig id="fig2s1" position="float" specific-use="child-fig">
                  <label>Figure 2—figure supplement 1.</label>
                  <caption>
                  <title>The representation of the Major Subject Areas, Research Organisms and author keywords on the eLife HTML page</title>
                  </caption>
                  <graphic mimetype="image" mime-subtype="tiff" xlink:href="elife-00666-fig2-figsupp1.tif"/>
         </fig>
 <fig-group>
```
The content of the `label` is determined by the `label` of the main `fig`, and the position of the figure supplement.
If the main `fig` has
```xml
<label>Appendix 1—figure 1.</label>
```
then it's first figure supplement label would be
```xml
<label>Appendix 1—figure 1—figure supplement 1.</label>
```
In other words, the figure supplement label is always prefixed by the label of the main figure.

# Mock ups

# Proposal
